### ISSUE TITLE
How to apply same style of OSM NY on some other GeoPackage

### ISSUE THREAD

**Issue by mohsan-hassan** [geosolutions-it/osm-styles#26]
hello,  I have downloaded your provided data directory and everything is working perfect as given, low res world map and detailed map of NY city with each and every item is displayed as per zoom level,

i want the same OSM style with some other country, I have downloaded the OSM (shape files) from https://download.geofabrik.de/ and converted those shape files into GeoPackage using QGIS,

After importing this GeoPackage in GeoServer via Store, I published one of the layer of type lines (roads) and applied osm:roads style to that layer, but it does not work and gives me error 
 The requested Style can not be used with this layer.  The style specifies an attribute of access and the layer is: xxxxxxx

In short I want to apply same OSM style to the shape files downloaded from OSM source, can you guide me

**Comment by jamesfry** [comment]
I imagine you need to update the styles to match the schemas in the Geofabrik exports.

You may have less work / more luck importing the .pbf of the same areas into a Postgres database using the imposm3 mappings supplied in the repo (https://github.com/geosolutions-it/osm-styles/tree/master/imposm) into a postgis instance and using that. If you need a geopackage solution then you can export the postgis database to a geopackage.

**Comment by aaime** [comment]
Indeed, GeoPackage sources are not supported, please follow the guide in the readme and use imposm instead.
(using GeoPackage may be possible, but the data dir needs to be reconfigured to support it)
